src/App.vue

```diff
@@ -1,138 +1,54 @@
+<!-- eslint-disable linebreak-style -->
+<script setup lang="ts">
+import {ref} from 'vue';
+import {useRouter} from 'vue-router';
+
+const drawer = ref(false);
+
+const router = useRouter();
+
+</script>
+
+
 <template>
   <v-app>
-    <v-app-bar>
+    <v-app-bar :elevation="0">
       <template #prepend>
         <v-app-bar-nav-icon
-          icon="mdi:menu"
-          @click.stop="toggleNav()"
+          id="drawer-icon"
+          @click="drawer = !drawer"
         />
       </template>
-
-      <v-app-bar-title id="title">
-        {{ getTitle }}
-      </v-app-bar-title>
-
-      <template #append>
-        <v-switch
-          v-model="powerModel"
-          inset
-          true-value="on"
-          false-value="off"
-          true-icon="mdi:power-on"
-          false-icon="mdi:power-off"
-        >
-          <template #label>
-            <v-icon
-              icon="mdi:power"
-              :style="powerModel == `on` ? 'color: green;' : 'color: red;'"
-            />
-          </template>
-        </v-switch>
-      </template>
+      <v-app-bar-title>Home</v-app-bar-title>
     </v-app-bar>
-    <v-navigation-drawer v-model="showNavBar">
+    <v-navigation-drawer v-model="drawer">
       <v-list nav>
         <v-list-item
-          v-for="(item, i) in getNavList"
-          :key="i"
-          :to="item.to"
-        >
-          <template #prepend>
-            <v-icon :icon="item.icon"/>
-          </template>
-          <v-list-item-title v-text="item.title"/>
-        </v-list-item>
+          title="home"
+          value="home"
+          @click="router.push('/'); drawer = !drawer"
+        />
+        <v-list-item
+          title="about"
+          value="about"
+          @click="router.push('/about'); drawer = !drawer"
+        />
       </v-list>
     </v-navigation-drawer>
     <v-main>
-      <router-view/>
+      <RouterView/>
     </v-main>
-    <v-footer height="25">
```

Comment:
This section here has been removed, but this is the theme selector switch

Reply:
I've just noticed that #8 does remove some of that so I'll have another look once that's merged